### PR TITLE
fix #62241 indianexpress.com

### LINKED ISSUE #62241: indianexpress.com
### Issue URL (Annoyance)
 [https://indianexpress.com/article/india/coronavirus-india-live-updates-maharashtra-delhi-andhra-pradesh-6566970/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Findianexpress.com%2Farticle%2Findia%2Fcoronavirus-india-live-updates-maharashtra-delhi-andhra-pradesh-6566970%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Content Blocker v2.6.2
Browser: | Samsung Browser
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#62241

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -202,6 +202,7 @@ walletinvestor.com#$#body { overflow: visible!important; }
 academyfx.ru#$#.modal-sale { display: none !important; }
 academyfx.ru#$#body { overflow: visible !important; }
 academyfx.ru#$#html { overflow: visible !important; }
+indianexpress.com##body > div#append_breaking_box
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:205]: Not sure if it must be blocked.

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -2590,6 +2591,7 @@ saocarlosagora.com.br#$?#html.fancybox-lock:has(> body) { overflow: visible!impo
 m.hh.ru##div[data-qa$="-banner"]
 !+ PLATFORM(android, ios, ext_android_cb)
 maxfoodfun.com##.pixnet-happix-pilot__launcher-bar__gift-notification
+indianexpress.com###append_cubebox_box
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:2594]: As previous. Better hide `#append_breaking_box`

File: SocialFilter/sections/popups.txt [hunk 3]
```diff
@@ -388,6 +388,8 @@ hardreset.info#$##modal-facebook { display: none !important; }
 hardreset.info#$#.modal-backdrop { display: none !important; }
 hardreset.info#$#body { overflow: visible!important; }
 !
+!+ PLATFORM(ios, ext_android_cb, android)
```
Comment (Alex-302) [inline SocialFilter/sections/popups.txt:391]: why for these platforms?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/popups.txt:205]: Same element with id="append_breaking_box" is pretty annoying on mobile layot.

File: SocialFilter/sections/popups.txt (hunk 3 above)
Comment (stanislav-atr) [reply SocialFilter/sections/popups.txt:391]: `.social-icon ` bar only appears for mobile layout.

File: AnnoyancesFilter/sections/popups.txt (hunk 2 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/popups.txt:2594]: Thats actually the element that was reported in the issue, floating animated popup

Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt:2594]: Not all reported content must be blocked.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**